src/css/nav.css

```
@@ -390,3 +686,893 @@ a.nav-text {
+  right: 0;
+  min-width: 120px;
+  padding: 6px;
+  background: #1a2332;
+  border: 1px solid rgba(255, 255, 255, 0.08);
+  border-radius: 8px;
+  box-shadow: 0 8px 24px rgba(0, 0, 0, 0.32);
+  z-index: 200;
+  animation: bucketVersionMenuIn 0.15s ease-out;
+}
+
+@keyframes bucketVersionMenuIn {
```

Stylelint (17.11.1)

[error] Expected keyframe name "bucketVersionMenuIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)